theseus/embodied/collision/tests/utils.py

```diff
@@ -0,0 +1,35 @@
+import torch
```

Comment:
Missing license header. Also `theseus/embodied/collision/tests/__init__.py` is missing license hearder.

Reply:
Good catch! Fixed.